packages/block-editor/src/hooks/block-hooks.js

```diff
@@ -201,6 +201,7 @@ function BlockHooksControlPure( {
 
 								return (
 									<ToggleControl
+										__nextHasNoMarginBottom
```

Comment:
## Testing instructions

Apply the following diff. In the Site Editor, go to Templates ▸ Single Posts template. See the block inspector for the Content block.

```diff
diff --git a/gutenberg.php b/gutenberg.php
index b12c346741..fe14f2cafb 100644
--- a/gutenberg.php
+++ b/gutenberg.php
@@ -74,3 +74,14 @@ function gutenberg_pre_init() {
 
 	require_once __DIR__ . '/lib/load.php';
 }
+
+add_filter( 'hooked_block_types', function ( $hooked_blocks, $position, $anchor_block ) {
+	if ( $anchor_block === 'core/post-content' && $position === 'after' ) {
+		// Add some core blocks after the post content.
+		$hooked_blocks[] = 'core/paragraph';
+		$hooked_blocks[] = 'core/separator';
+		$hooked_blocks[] = 'core/page-list';
+	}
+
+	return $hooked_blocks;
+}, 10, 3 );

```

<img src="https://github.com/user-attachments/assets/cadc8385-56f0-4a50-91ed-d976d766007a" alt="ToggleControls in Content block inspector" width="278">